app/Mage.php

```diff
@@ -863,7 +863,14 @@ public static function log($message, $level = null, $file = '', $forceLog = fals
 
         static $loggers = array();
 
+        $maxLogLevel = (int) self::getStoreConfig('dev/log/max_level');
```

Comment:
this should be wrapped in try / catch because getStoreConfig can throw several exceptions (especially is Mage::log is used during early  stage of the request flow.

Reply:
Thanks! PR created.